src/Containers/FeaturedProfilesCard.tsx

```diff
@@ -21,140 +22,118 @@ export interface IFeaturedProfilesCardProps {
-            MAX_PROFILES,
+            determineMaxProfileLength(),
         );
 
-        const userProfilePicture = (profile: IProfile): React.ReactElement => {
-            return (
-                <CircleImage background="none" key={profile.id}>
-                    <img
-                        src={profile.image}
-                        alt={alt}
-                        height={height}
-                        width={width}
-                    />
-                </CircleImage>
-            );
-        };
-
-        const userInitials = (profile: IProfile): React.ReactElement => {
-            return (
-                <CircleImage background="orange" key={profile.id}>
-                    <CircleContent>{profile.initials}</CircleContent>
-                </CircleImage>
-            );
-        };
+        const profileInitialsEndIndex =
+            profiles.length - PROFILE_INITIALS_INDEX_ENDPOINT;
 
-        const remainingUserProfiles = (
+        const getProfiles = (
             profile: IProfile,
+            index: number,
         ): React.ReactElement => {
-            const remaining_profiles_value =
-                profileData.length - REMAINING_PROFILES_INDEX;
-
-            return (
-                <CircleImage background="grey" key={profile.id}>
-                    <CircleContent>{remaining_profiles_value}</CircleContent>
-                </CircleImage>
-            );
-        };
-
-        const addUserIcon = (profile: IProfile): React.ReactElement => {
-            return (
-                <CircleImage
-                    key={profile.id}
-                    background="grey"
-                    onClick={(): void => onClick(profile)}
-                >
-                    <Icon as={AddUser} />
-                </CircleImage>
-            );
+            switch (true) {
+                case index < getProfilePictureLimit():
+                    return (
+                        <FeaturedProfile profile={profile} background="none">
+                            <img
+                                src={profile.image}
+                                alt={alt}
+                                height={height}
+                                width={width}
+                            />
+                        </FeaturedProfile>
+                    );
+                case index < profileInitialsEndIndex:
+                    return (
+                        <FeaturedProfile profile={profile} background="orange">
+                            <CircleContent>{profile.initials}</CircleContent>
+                        </FeaturedProfile>
+                    );
+                default:
+                    return (
+                        <FeaturedProfile profile={profile} background="gray">
+                            <CircleContent>
+                                {profileData.length - profileInitialsEndIndex}
+                            </CircleContent>
+                        </FeaturedProfile>
```

Comment:
FeatureProfile is repeated here three times. You are returning either an image or a circle content, FeatureProfile can handle this through props. In the end this will look like <FeaturedProfile whateverprops />. instead of this whole thing repeated three times.